include/rmm/cuda_device.hpp

```diff
@@ -26,16 +26,20 @@ namespace rmm {
  *
  */
 struct cuda_device_id {
-  using value_type = int;
+  using value_type = int;  ///< Integer type used for device identifier
 
   /**
    * @brief Construct a `cuda_device_id` from the specified integer value
    *
-   * @param id The device's integer identifier
+   * @param dev_id The device's integer identifier
    */
   explicit constexpr cuda_device_id(value_type dev_id) noexcept : id_{dev_id} {}
 
-  /// Returns the wrapped integer value
+  /**
+   * @brief Returns the wrapped integer value
+   *
+   * @return The wrapped integer value
```

Comment:
Hmmm, actually the @brief and the @return do the same thing. Can we just drop the @return and have the @brief. I like just having @brief for simple getters.